pos_cash_move_reason/wizard/pos_box.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="cash_box_in_form">
+            <field name="name">cash.box.in.form</field>
```

Comment:
FYI, this line is not necessary. if the name is not set, the name will be autogenerated with model.name + form type. (here, it will be `cash.box.in.form`)
cosmectic remark, not a blocking point.